test/test_websocket.py

```diff
@@ -484,10 +484,13 @@ def bad_completion_callback(data):
             # wait for shutdown...
             handler.shutdown_future.result(TIMEOUT)
 
+            # Change in this PR in aws-c-io https://github.com/awslabs/aws-c-io/pull/650
+            # along with this PR in aws-c-http https://github.com/awslabs/aws-c-http/pull/474
+            # allows a window update during shutdown.
```

Comment:
hmmm, how is this related to window update during shutdown? It seems sending data after shutdown to me?

Reply:
Restoring test thanks to https://github.com/awslabs/aws-c-http/pull/483#pullrequestreview-2240952701